lib/json/store.nit

```diff
@@ -111,7 +111,7 @@ class JsonStore
 
 	# Is there data are stored under `key`.
 	fun has_key(key: String): Bool do
-		return (store_dir / "{key}.json").file_exists
+		return ("{store_dir}/{key}.json".simplify_path).file_exists
```

Comment:
I'm not sure I get it.

~~~
$ nit -e 'print ".github_data" / "foo" '
.github_data/foo
~~~